app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,4 @@
 <resources>
-    <string name="app_name">IChat</string>
+    <string name="app_name">Personal Chat</string>
```

Comment:
The name seems not correct, Just make it Chat App.

Reply:
Change been done as requested.